python/enthic/compute_stats.py

```diff
@@ -0,0 +1,211 @@
+# -*- coding: utf-8 -*-
+"""
+=========================================================================
+Compute statistics from companies's data after complete and check it
+=========================================================================
+
+PROGRAM BY PAPIT SASU, 2019
+
+Coding Rules:
+
+- Snake case for variables.
+- Only argument is configuration file.
+- No output or print, just log and files.
+"""
+import json
+import math
+
+def convert_data_to_tree(rawData):
```

Comment:
Please use snake case for all variable as stated in PEP8 https://www.python.org/dev/peps/pep-0008/#id45